src/eko/anomalous_dimensions/__init__.py

```diff
@@ -127,41 +130,55 @@ def gamma_ns(order, mode, n, nf):
         elif mode == 10200:
             gamma_ns_2 = -as3.gamma_nsv(n, nf, sx)
         gamma_ns[2] = gamma_ns_2
+    # N3LO
+    if order[0] >= 4:
+        sx = harmonics.compute_harmonics_cache(n, 5, is_singlet=False)
```

Comment:
Can we reuse the already computed `sx` here? (since I guess these additional ones will need them in one way or the other)

Reply:
yes this is possible, will fix it.